### ISSUE TITLE
Plugin unable to launch or install twitch games

### ISSUE THREAD

**Issue by r0mnic** [nyash-qq/galaxy-plugin-twitch#35]
The plugin is able to connect to my twitch app, and lists all my twitch games, but installing twitch games or trying to launch games I previously installed opens a pop-up saying "You'll need a new app to open this twitch link"

![Screenshot (1)](https://user-images.githubusercontent.com/68295071/87456674-4421e780-c5d5-11ea-8b03-7bedaa251ba6.png)

**Comment by thespyman13** [comment]
all twitch games have been moved to Amazon Games App, required to play now.

**Comment by Rall3n** [comment]
> all twitch games have been moved to Amazon Games App, required to play now.

Wrong! The games are still downloadable through the Twitch Desktop App. They just advertise it less prominently.

![image](https://user-images.githubusercontent.com/15950630/90281516-fc5cdd00-de6c-11ea-8c9c-e8c34a827ab4.png)

-----

@jpegry Seems like there is no scheme handler registered for the Twitch url scheme (`twitch://`).

Please check if the registry entry under the key `Computer\HKEY_CLASSES_ROOT\twitch` exists and if the subkey `shell\open\command` (`Computer\HKEY_CLASSES_ROOT\twitch\shell\open\command` in full) has a value with a path to your Twitch application. If not, then you could try adding it yourself or try reinstalling the application.

Value should look like this:
```
"C:\Users\USERNAME\AppData\Roaming\Twitch\Bin\Twitch.exe" -- "%1"
```
